src/frontend/src/lib/config/send.config.ts

```diff
@@ -18,9 +13,21 @@ export const sendWizardSteps = (i18n: I18n): WizardSteps => [
 	{
 		name: WizardStepsSend.SENDING,
 		title: i18n.send.text.sending
-	},
+	}
+];
+
+export const sendWizardStepsWithQrCodeScan = (i18n: I18n): WizardSteps => [
+	...sendWizardSteps(i18n),
```

Comment:
Why no TOKENS_LIST here?

Reply:
this is used by the `EthSendModal` and the `IcSendModal`, so we would need an object that has all steps except the tokens list. We can invert and create one that has all and the other one that filters out the TOKENS_LIST step

Reply:
Ahhh now I get it. All good.